views/beertarget.hbs

```diff
@@ -0,0 +1,9 @@
+{{#each resultBeer}}
+
+<div>
```

Comment:
Same issue with indentation. For consistency, have all your indentations set at 2. Right now there are some indentations at 2 and some at 4 in different pages. You can change that in the vs code settings, click on the botton right where it says Spaces: and set it to be 2 in all pages.